src/main/app/src/app/taken/taken-vrijgeven-dialog/taken-vrijgeven-dialog.component.html

```diff
@@ -0,0 +1,20 @@
+<!--
+  ~ SPDX-FileCopyrightText: 2021 Atos
+  ~ SPDX-License-Identifier: EUPL-1.2+
+  -->
+
+<h1 mat-dialog-title>{{'title.taken.vrijgeven' | translate}}</h1>
+
+<div mat-dialog-content>
+    <p *ngIf="data.length == 1">{{'msg.vrijgeven.taak' | translate }}</p>
+    <p *ngIf="data.length != 1">{{ 'msg.vrijgeven.taken' | translate:{aantal: data.length} }}</p>
+</div>
+<div mat-dialog-actions>
+    <button [disabled]="loading" mat-button color="primary" (click)="vrijgeven()">
```

Comment:
Ik mis hier nog een id, het format dat we aanhouden is entiteitActie_attribuut dus dat zou dan taakVrijgeven_button worden.